File: push/src/main/java/com/xiaomi/push/service/PacketSendCallback.java
package com.xiaomi.push.service;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/push/service/PacketSendCallback.class */
public interface PacketSendCallback {
    void onAnswer(String str, String str2, String str3, String str4);

    void onFail(PacketSendFailCause packetSendFailCause, String str);

    void onSent();
}

File: push/src/main/java/com/xiaomi/push/service/PacketSendCallback.kt
package com.xiaomi.push.service

interface PacketSendCallback {
    fun onAnswer(str: String, str2: String, str3: String, str4: String)

    fun onFail(packetSendFailCause: PacketSendFailCause, str: String)

    fun onSent()
}
